oauth2/fosite_store_sql.go

```diff
@@ -87,6 +87,8 @@ func sqlSchemaUp(table string, id string) string {
 	subject 		varchar(255) NOT NULL
 )`,
 		"4": fmt.Sprintf("ALTER TABLE hydra_oauth2_%s ADD active BOOL NOT NULL DEFAULT TRUE", table),
+		"5": fmt.Sprintf("CREATE INDEX hydra_oauth2_%s_request_id_idx ON hydra_oauth2_%s (request_id)", table, table),
```

Comment:
I think it would make sense to have an unique index. They key should really be unique and it will help with query speed.

Reply:
Nice catch 👍

Reply:
lmao 2 years later and I found that this new unique index actually breaks OpenID Connect compliance...welp 🤷